flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOSecuredITCase.java

```diff
@@ -97,6 +98,15 @@ public static void teardownSecureCluster() throws Exception {
 		SecureTestEnvironment.cleanup();
 	}
 
+	@Override
+	public void testDetachedMode() throws InterruptedException, IOException {
+		super.testDetachedMode();
+		ensureStringInNamedLogFiles(new String[]{"Login successful for user", "using keytab file"},
```

Comment:
From which component do these logging statements come from?

Reply:
Please see UserGroupInformation.loginUserFromKeytab().

Reply:
Thanks for the pointer :-)